src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -122,8 +123,12 @@ public RubyContext(RubyLanguage language, TruffleLanguage.Env env) {
                     "this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
         }
 
-        rubyHome = findRubyHome();
-        Log.LOGGER.config(() -> String.format("ruby home: %s", rubyHome));
+        try {
+            rubyHome = findRubyHome();
+        } catch (IOException e) {
+            throw new JavaException(e);
+        }
+        Log.config("ruby home: %s", rubyHome);
```

Comment:
Ah, did not know about that option, will fix.
We should have a test for it if it matters.